src/core/utils/assets.ts

```diff
@@ -100,18 +100,16 @@ export function parseAsset({
 }): ParsedAsset {
   const address = asset.asset_code;
   const chainName = asset.network ?? ChainName.mainnet;
+  const networks = 'networks' in asset ? asset.networks || {} : {};
   const chainId =
-    chainIdFromChainName(chainName) ||
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    (Number(Object.keys(asset.networks)?.[0]) as ChainId);
+    chainIdFromChainName(chainName) || Number(Object.keys(networks)[0]);
```

Comment:
some assets don't return a `networks` this was erroring in some custom networks and making so that nothing was shown in the activity tab, Im talking with Benjamin to find a better solution

Reply:
we got new updates from backend  now, would it fix this?

Reply:
yes updated to use the new `chain_id` field from the api response